nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -164,6 +164,8 @@ Use `configure.packages` instead.
 
 - The `pass-secret-service` package now includes systemd units from upstream, so adding it to the NixOS `services.dbus.packages` option will make it start automatically as a systemd user service when an application tries to talk to the libsecret D-Bus API.
 
+- The `heroic` package is now wrapped in a FHS environment so that [third-party Wine builds](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/How-To:-Wine-and-Proton) (such as Wine-GE) will work.  The bare `heroic` package is named `heroic-unwrapped`.
+
```

Comment:
Is that so note worthy? I wouldn't have created a changelog entry here.

Reply:
I was unsure, so I thought it better to add one.  I shall drop it.